### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by francksefu**
Add features of the desktop screen in the CSS file 
upload background images for the desktop screen
hide the navigation bar of the mobile screen and set another navigation for the desktop screen

### CODE REVIEW

File: styles2.css [hunk 1]
```diff
@@ -657,3 +542,267 @@ footer > div.contentline {
...
+    display: grid;
+    grid-template-columns: 369px 369px 369px;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: space-between;
+    row-gap: 24px;
+    column-gap: 32px;
+    margin-bottom: 28px;
+  }
```
Comment (AlphaNtihinduka) [inline styles2.css]: - [ ] You have done great implementing the cards using `grid` :+1: . However, the recent works section is not responsive for widths below `1244px`. This leads to a `horizontal scroll` Kindly ensure that the width reduces as the responsiveness width reduces. You may consider the use of `max-width` or `percentage width`.
-Your design :sunglasses: 
Screenshot from 2023-01-20 06-14-49

- Figma design :sunglasses: 
Screenshot from 2023-01-20 06-12-41

File: styles2.css [hunk 2]
```diff
@@ -657,3 +542,267 @@ footer > div.contentline {
...
+
+  .icon.desktop > ul {
+    margin: 15px auto;
+    width: 100%;
+  }
+
+  hr {
+    height: 1px;
+    color: #dfe1e6;
+  }
```
Comment (AlphaNtihinduka) [inline styles2.css]: - [ ] I have noticed that the `hr` does not cover `100% width` of the `body` :disappointed: . You should ensure that the line sticks to the width of the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=Kv8PktMsmpk2hK5U-0). :+1: 
- Your design :sunglasses: 
Screenshot from 2023-01-20 06-34-29

File: index2.html [hunk 3]
```diff
@@ -309,10 +309,37 @@ <h3>Skills</h3>
...
+                    <a href="https://www.linkedin.com/in/franck-sefu-884705254/">
+                        <i class="fab fa-linkedin-in"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/francksefu">
+                        <i class="fab fa-github"></i>
+                    </a>
+                </li>
+            </ul>
```
Comment (AlphaNtihinduka) [inline index2.html]: - [ ] The Figma design proves that there is an icon that you are missing in your design. This icon should be included in the footer to stick to the Figma design and have a professional website. :+1: 
- Your design :sunglasses: 
Screenshot from 2023-01-20 06-38-05

- Figma design :sunglasses: 
Screenshot from 2023-01-20 06-38-51

File: styles2.css [hunk 4]
```diff
@@ -657,3 +542,267 @@ footer > div.contentline {
...
+  .contact-form {
+    display: grid;
+    grid-template-columns: 436px 1fr;
+    align-items: flex-start;
+    padding: 80px 0;
+    gap: 24px;
+    width: 1171px;
+    margin: 0 auto 145px auto;
+    background-image: none;
+  }
```
Comment (AlphaNtihinduka) [inline styles2.css]: - [ ] The contact form should not create a `horizontal scroll` at any `breakpoint` :exclamation: . You should ensure that the width of the form section is responsive to stick to the Figma design. This makes the website look professional :+1: 
- Your design :sunglasses: 
Screenshot from 2023-01-20 06-47-19

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @francksefu     🙋‍♂️
Good job working on implementing the requirements of the desktop version. You have done the design to the best expectation and incredibly well. 👏  
However, there are some issues mostly with the `horizontal scroll` effect.
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
- Kindly consider the inline comments for the changes you should apply to have the project approved.
- [IMPORTANT TO NOTE] Please ensure that all the horizontal scroll effects on every section are fixed to make the project look professional. This increases the chances of being considered by the recruiters :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- most of changes implemented ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  As mentioned by the previous reviewer kindly fix the horizontal scroll for your website _**

scroll





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @francksefu , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it :ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Good commit messages.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**